flink-tests/src/test/java/org/apache/flink/test/checkpointing/RegionFailoverITCase.java

```diff
@@ -79,11 +82,15 @@
 import java.util.stream.IntStream;
 import java.util.stream.StreamSupport;
 
+import static org.apache.flink.runtime.dispatcher.SchedulerNGFactoryFactory.SCHEDULER_TYPE_LEGACY;
+import static org.apache.flink.runtime.dispatcher.SchedulerNGFactoryFactory.SCHEDULER_TYPE_NG;
+import static org.apache.flink.util.Preconditions.checkNotNull;
 import static org.junit.Assert.assertEquals;
 
 /**
  * Tests for region failover with multi regions.
  */
+@RunWith(Parameterized.class)
 public class RegionFailoverITCase extends TestLogger {
```

Comment:
We need to annotate it with `AlsoRunWithSchedulerNG` category.

Reply:
I don't think it's needed here because it wouldn't add test coverage. We already always run this test with both schedulers independent of the profile.

Reply:
Ah, my dumb mistake.